src/types/publication.types.ts

```diff
@@ -28,7 +36,7 @@ export interface Publisher {
   title: string;
   printIssn: string;
   onlineIssn: string;
-  level: number | null;
+  level: string | number | null;
```

Comment:
Må fortsatt ha med number siden vi bruker samme type for oppslag mot kanalregister?

Reply:
ja, første gang vi lagrer er det `number | null`, mens etter vi har lagret en gang (eller hvis vi laster inn en publikasjon) så er det en `string`.

Reply:
så det kan fortsatt stå `mixed` som type i Yup imo

Reply:
ja, du kan sikkert bare fjerne den TODO-kommentaren på mixed med det samme da